cypress/lib/url.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Generate a full URL for a given relative path and the desired stage
+ *
+ * @param {'dev' | 'code' | 'prod'} stage
+ * @param {string} path
+ * @param {{ isDcr?: boolean }} options
+ * @returns {string} The full path
+ */
+export const getTestUrl = (stage, path, { isDcr } = { isDcr: false }) => {
+	switch (stage?.toLowerCase()) {
```

Comment:
very minor: it might be worth sanitizing the string as soon as possible i.e. after we call `Cypress.env`

Reply:
Good shout!

Reply:
Have updated the PR